README.md

```diff
@@ -24,18 +24,29 @@ pip install pybravia
 
 ```py
 from pybravia import BraviaClient, BraviaError
+import asyncio
```

Comment:
Changes to the README are simply to make this example runnable. I couldn't run the current form because the async wasn't being applied to a function.